com.remnote.RemNote.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.remnote.RemNote</id>
+  <launchable type="desktop-id">com.remnote.RemNote.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
link the licence?

Reply:
I couldn't find one on their site or forum. Will the following work?

https://www.remnote.com/terms_of_service

Reply:
ok